libevmasm/Assembly.cpp

```diff
@@ -74,6 +74,117 @@ unsigned Assembly::codeSize(unsigned subTagSize) const
 	}
 }
 
+void Assembly::addAssemblyItemsFromJSON(Json::Value const& _code)
+{
+	solAssert(_code.isArray(), "");
```

Comment:
Should this also assert that `m_items` is empty?

Reply:
We could do this. Maybe it's safer if we assert `m_items` is empty here. I think we don't really need to be able to add assembly items from json multiple times, and if we need that in the future, we can just remove the assert.